foundation/core/variables.tf

```diff
@@ -115,3 +119,58 @@ variable "enable_point_to_site_vpn" {
   description = "If enabled, creates User Point to Site VPN configuration in CAF foundation."
   default     = true
 }
+
+variable "slack_func_identifier" {
+  type    = string
+  default = "billing_alert_func"
+}
+
+variable "provision_budget_alerts" {
```

Comment:
Typically, root modules that have a flag like this will just call it "enabled".  So you can set `enabled = false` if you want to disable it.